docs/upgrade_notes.rst

```diff
@@ -16,6 +16,17 @@ Libcloud 0.20.0
   to the constructor (you are encouraged to use keyword arguments to avoid such
   issues in the future).
 
+* All NodeState, StorageVolumeState, VolumeSnapshotState and Provider attributes
+  are now strings instead of integers.
+
+  If you are using the ``tostring`` and ``fromstring`` methods of NodeState,
```

Comment:
It would also be good to note that you are affected if you directly store the enum value in the database (integer) - I know some users who do that.

For that scenario this is a backward incompatible change. We don't have any automatic migration path for it - users need to handle that in their code - so for users who do that it's quite an invasive change.